docs/root/configuration/overview/xds_api.rst

```diff
@@ -348,3 +348,25 @@ To use delta, simply set the api_type field of your
 That works for both xDS and ADS; for ADS, it's the api_type field of
 :ref:`DynamicResources.ads_config <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.dynamic_resources>`,
 as described in the previous section.
+
+
+.. _config_overview_ttl:
+
+xDS TTL
+^^^^^^^
+
+When using xDS, users might find themself wanting to temporarily update certain xDS resources. In order to do
+safely, xDS TTLs can be used to make sure that if the control plane becomes unavailable and is unable to revert
+the xDS change, Envoy will remove the resource after a TTL specified by the server. See the
+:ref:`protocol documentation <xds_protocol_ttl>` for more information.
+
+Currently the behavior when a TTL expires is that the resource is *expired* (as opposed to reverted to the
```

Comment:
Maybe change "the resource is *expired*" to "the resource is *removed*"?  I'm thinking that might make the meaning a bit clearer.